Document/0x06b-Basic-Security-Testing.md

```diff
@@ -529,6 +529,8 @@ root@localhost's password:
 iPhone:~ root#
 ```
 
+You can [make the iproxy run automatically in the background](https://iphonedevwiki.net/index.php/SSH_Over_USB "Making iproxy run automatically in the background on OS X") if you don't want to run the binary every time you want to SSH over USB.
```

Comment:
could you please move this line so that the text looks like this?

The above command maps port `22` on the iOS device to port `2222` on localhost. You can also [make iproxy run automatically in the background](https://iphonedevwiki.net/index.php/SSH_Over_USB "Making iproxy run automatically in the background on OS X") if you don't want to run the binary every time you want to SSH over USB.

With the following...